starter/tsconfig.json

```diff
@@ -12,7 +12,9 @@
     ],
     "paths": {
       "@angular/*": ["../node_modules/@angular/*"],
-      "@nationalbankbelgium/*": ["../node_modules/@nationalbankbelgium/*"]
+      "@nationalbankbelgium/*": ["../node_modules/@nationalbankbelgium/*"],
+      "core-js/es7/reflect": ["../node_modules/core-js/proposals/reflect-metadata"],
+      "core-js/es6/*": ["../node_modules/core-js/es/*"]
```

Comment:
are these needed? because it seems you removed them from `starter/src/polyfills.browser.ts`

Reply:
In fact this is the workaround to get Angular CLI working with the latest `core-js`... See https://github.com/angular/angular-cli/issues/13954... I would like to add a FIXME somewhere to mention this but where do I add it? This is a json file and we cannot put comments in here :(